packages/cactus-test-tooling/src/test/typescript/integration/besu/besu-test-ledger/constructor-validates-options.ts

```diff
@@ -7,27 +7,22 @@ import {
   isIKeyPair,
 } from "../../../../../main/typescript/public-api";
 
-tap.test("constructor throws if invalid input is provided", (assert: any) => {
+test("constructor throws if invalid input is provided", (assert: Test) => {
   assert.ok(BesuTestLedger);
   assert.throws(() => new BesuTestLedger({ containerImageVersion: "nope" }));
   assert.end();
 });
 
-tap.test(
-  "constructor does not throw if valid input is provided",
-  (assert: any) => {
-    assert.ok(BesuTestLedger);
-    assert.doesNotThrow(() => new BesuTestLedger());
-    assert.end();
-  },
-);
+test("constructor does not throw if valid input is provided", (assert: Test) => {
+  assert.ok(BesuTestLedger);
+  assert.doesNotThrow(() => new BesuTestLedger());
+  assert.end();
+});
 
-tap.test("starts/stops/destroys a docker container", async (assert: any) => {
+test("starts/stops/destroys a docker container", async (assert: Test) => {
   const besuTestLedger = new BesuTestLedger();
-  assert.tearDown(async () => {
-    await besuTestLedger.stop();
-    await besuTestLedger.destroy();
-  });
+  test.onFinish(() => besuTestLedger.stop());
+  test.onFinish(() => besuTestLedger.destroy());
```

Comment:
@awadhana There seems to be a race condition here, I recommend ensuring that the stop method finishes before the destroy is called (otherwise the destroy method might fail)

```suggestion
  test.onFinish(async () => {
    await besuTestLedger.stop();
    await besuTestLedger.destroy();
  });
```